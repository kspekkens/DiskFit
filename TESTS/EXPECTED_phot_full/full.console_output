 DiskFit (v1.2.2)  Copyright (C) 2015, 2017, Jerry Sellwood and Kristine Spekkens

 This program comes with ABSOLUTELY NO WARRANTY.  It is free software
 and you are welcome to redistribute it under certain conditions.
 See <http://www.gnu.org/licenses/> for details.

 Enter input parameter file name

 Read in input parameters from: EXAMPLE/PHOT/phot_full.inp                                                                          

Input disk eps: (1-b/a)  0.40
Input disk PA (N->E; deg) =  35.00
Input x-center (data units) =   130.00
Input y-center (data units) =   130.00
Sky, sky sig, gain (ADU) =   800.00   14.14    4.00

 Fitting for:
 disk PA
 disk ellipticity
 disk x-center, y-center
 Bar profile
 Bar PA
 Bar ellipticity
 Sersic bulge profile
 bulge effective radius
 bulge ellipticity
 bulge Sersic index
 --> Other attributes fixed to input values

 Surface brightnesses extracted at radii:
    5.0   10.0   15.0   20.0   25.0   30.0   35.0   40.0   45.0   50.0
   55.0   60.0   65.0   70.0   75.0   80.0   90.0  100.0  110.0

 Fitting for a bar component
 Axisymmetric model outside    40.0000000    

 Read in photometric image from:
 EXAMPLE/PHOT/phot.fits

 Number of pixels in selected part of the image:       65536
 Number of pixels used in fit:        7495
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   35.0000  0.400000  130.0000  130.0000    5.0000    0.5000    0.2000    2.0000   10.0000
starting iteration    1 best reduced chisq so far: 1.838E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   35.0000  0.400000  130.0000  130.0000    5.0000    0.5000    0.2000    2.0000   10.0000
starting iteration    2 best reduced chisq so far: 9.880E-01
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.7303  0.306540  129.9455  129.9945   11.2600    0.5840    0.1452    1.9261   10.2436
starting iteration    3 best reduced chisq so far: 9.662E-01
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.7849  0.296538  129.9724  130.0020   10.1985    0.5922    0.1642    1.8599    9.8998
starting iteration    4 best reduced chisq so far: 9.644E-01
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   40.1262  0.295257  129.9756  130.0021    9.9673    0.5915    0.1769    1.7627    9.2936
starting iteration    5 best reduced chisq so far: 9.640E-01
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   40.0680  0.294488  129.9827  130.0015   10.0086    0.5911    0.1781    1.7385    9.1781
starting iteration    6 best reduced chisq so far: 9.637E-01
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.9360  0.293710  129.9916  130.0009   10.0593    0.5910    0.1789    1.6905    8.8266
starting iteration    7 best reduced chisq so far: 9.636E-01
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.9697  0.293546  129.9918  130.0008   10.0676    0.5910    0.1778    1.6857    8.7210
starting iteration    8 best reduced chisq so far: 9.636E-01
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.9823  0.292840  129.9915  130.0013   10.0497    0.5909    0.1739    1.6705    8.3468

 Done minimization
 Total number of iterations:            8
 Minimum chi^2 found:   0.963588774    

 Wrote:  EXAMPLE/PHOT/OUT/full.mod.fits
 Wrote:  EXAMPLE/PHOT/OUT/full.res.fits
 Wrote:  EXAMPLE/PHOT/OUT/full.dsk.fits
 Wrote:  EXAMPLE/PHOT/OUT/full.bar.fits
 Wrote:  EXAMPLE/PHOT/OUT/full.blg.fits
 Wrote out: EXAMPLE/PHOT/OUT/full.out
