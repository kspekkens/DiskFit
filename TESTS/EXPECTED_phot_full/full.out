Minimization output, phot
-------------------------
Input files:
EXAMPLE/PHOT/phot_full.inp
EXAMPLE/PHOT/phot.fits  pixscale  0.50 arcsec/pix
sky:  800.00  skysig:   14.14    gain:    4.00

FITS region:       x-low:      1   y-low:      1 x-range:    256 y-range:    256
FITS sampling:    regrad:  10.00   regpa:  35.00  regeps:   0.40 istpout:      2

Output model and (data-model) residuals files:
EXAMPLE/PHOT/OUT/full.mod
EXAMPLE/PHOT/OUT/full.res

Disk toggles:         PA: T     eps: T  center: T non-axi: T    phib: T
phot toggles:     lbulge: T    lr_e: T  lsersn: T blg eps: T bar eps: T

Input values
------------
disk PA, phi_d^prime (deg):           35.00
disk eps:                              0.40
x,y center (data units):             130.00  130.00
Bar PA (deg):                          5.00
Bar eps:                               0.50
Bulge r_e (pixels):                   10.00
Sersic n:                              2.00
Bulge eps:                             0.20




Seeing correction applied. FWHM (data units):      2.00
No model component smoothing applied
No uncertainties estimated
Non-axisymmetries fixed to 0 outside range:    5.00   40.00


Best fitting values
-------------------
disk PA, phi_d^prime (deg):           39.98 +/-  0.00
disk eps:                              0.29 +/-  0.00
disk incl (deg):                      44.99 +/-  0.00
x,y center (data units):             129.99 +/-  0.00,  130.00 +/-  0.00
Non-axisymm phib (sky plane, deg):    10.05 +/-  0.00, -129.15
Bar eps:                               0.59 +/-  0.00
Bulge r_e (pixels):                    8.35 +/-  0.00
Bulge I_0 (ADU):                     365.15 +/-     0.00
Sersic n:                              1.67 +/-  0.00
Bulge eps:                             0.17 +/-  0.00

Disk light fraction (%):              79.95 +/-  0.00
Bar light fraction (%):                9.84 +/-  0.00
Bulge light fraction (%):             10.21 +/-  0.00

Minimization Details
---------------------
# points Dn used in fit:               7495
# iterations in minimization:             8
Minimum chi^2 found:               0.963589
Degrees of freedom in fit:             7458

Fitted intensities                                         (radii in pixels, intensities in ADU):
       r        npts       Idisk      eIdisk        Ibar       eIbar     Ibulge      eIbulge
---------------------------------------------------------------------------------------------
      5.00    125.66      648.51        0.00      190.58        0.00      809.28        0.00
     10.00     99.62      634.22        0.00      173.98        0.00      258.82        0.00
     15.00     83.31      528.17        0.00      198.76        0.00      102.93        0.00
     20.00    111.42      442.70        0.00      207.67        0.00       46.06        0.00
     25.00    138.85      365.75        0.00      181.34        0.00       22.27        0.00
     30.00    167.81      297.14        0.00      247.87        0.00       11.40        0.00
     35.00    194.97      251.13        0.00       65.16        0.00        6.09        0.00
     40.00    222.14      200.74        0.00        0.00        0.00        3.37        0.00
     45.00    250.15      166.12        0.00        0.00        0.00        1.92        0.00
     50.00    277.37      134.53        0.00        0.00        0.00        1.12        0.00
     55.00    306.20      111.57        0.00        0.00        0.00        0.67        0.00
     60.00    333.80       89.43        0.00        0.00        0.00        0.41        0.00
     65.00    360.51       74.48        0.00        0.00        0.00        0.25        0.00
     70.00    388.49       60.16        0.00        0.00        0.00        0.16        0.00
     75.00    415.94       49.51        0.00        0.00        0.00        0.10        0.00
     80.00    680.90       38.74        0.00        0.00        0.00        0.06        0.00
     90.00    999.46       29.25        0.00        0.00        0.00        0.03        0.00
    100.00   1103.94        7.84        0.00        0.00        0.00        0.01        0.00
    110.00   1234.45        0.67        0.00        0.00        0.00        0.01        0.00
